src/NUnitEngine/nunit.engine/Extensibility/ExtensionNode.cs

```diff
@@ -128,7 +134,17 @@ public object ExtensionObject
         /// </summary>
         public object CreateExtensionObject(params object[] args)
         {
+#if NETSTANDARD2_0
+            var assembly = Assembly.LoadFrom(AssemblyPath);
+            var typeinfo = assembly.DefinedTypes.FirstOrDefault(t => t.FullName == TypeName);
+            if (typeinfo == null)
+            {
+                return null;
+            }
+            return Activator.CreateInstance(typeinfo.AsType(), args);
```

Comment:
Ref comment on ExtensionService - I think we might want to try/catch from here. (Also, I don't think this method is safe to return null - I think perhaps we should throw instead?)